### DISCUSSION TITLE
What are total segments, pageable, resident in Vertipaq Analyser

### DISCUSSION THREAD

**Discussion by capstoneanalytics** [DaxStudio/DaxStudio#1001] [Q&A] [answered]
In the new(ish) version of DAX studio there is extra information in the Vertipaq Analyser stats like Total segments, pageable, resident etc. I couldnt find anything about it in the documentation. Can you please explain what these are. Thanks.

Abhijith

**Comment by marcosqlbi** [comment, marked as answer]
The metrics reported by DAX Studio are related to information provided by VertiPaq Analyzer through DMVs (Data Management Views) and other measurements. Specifically, some of the metrics you mentioned are related to VertiPaq features (look for information/video/documents about how VertiPaq works) but others are not officially documented by Microsoft (pageable and resident, for example), even though they are exposed in DMVs.

**Reply by capstoneanalytics** [reply]
Thanks Marco !

**Comment by dgosbell** [comment]
Some of those columns are loosely documented here. https://learn.microsoft.com/en-us/openspecs/sql_server_protocols/ms-ssas/1079969e-b432-48fc-bff7-20ced3a96893 but they don't really go into much detail.

**Reply by capstoneanalytics** [reply]
Thanks Darren !. That is a very hand link.
